### PR TITLE
Bootstrapping of zero point and zero point error

### PR DESCRIPTION
**Pull request by emirkmo**
This pull request is for adding the bootstrapping of zero point and zero point error calculations. Currently, the simple LSQ and bootstrapping are both done, and recorded in the meta header. A logger warning is raised if the zero points estimated by these methods are significantly different. My testing on a few images has found this not to be the case.

The sigma for the sigma_clipping is calculated with [Chauvenet's criteria](https://en.wikipedia.org/wiki/Chauvenet%27s_criterion), but is a minimum of 1.5 (incase sample size is low).

This PR addresses #9

### CODE REVIEW

**Comment by emirkmo** [comment]
@rhandberg  Do you think we should change the sigma calculation for the least square fitting method to the one from Chauvenet's criteria as well?.

File: flows/photometry.py [hunk 1]
```diff
@@ -617,8 +644,10 @@ def photometry(fileid, output_folder=None, attempt_imagematch=True):
 	tab.meta['pixel_scale'] = pixel_scale * u.arcsec/u.pixel
 	tab.meta['seeing'] = (fwhm*pixel_scale) * u.arcsec
 	tab.meta['obstime-bmjd'] = float(image.obstime.mjd)
-	tab.meta['zp'] = zp
-	tab.meta['zp_error'] = zp_error
+	tab.meta['zp'] = zp_bs
```
Comment (rhandberg) [inline flows/photometry.py:647]: May I suggest to not add BOTH of them to the headers... I think it will just be a source of confusion... "Which one was actually used?"

File: flows/zeropoint.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Plotting utilities.
```
Comment (rhandberg) [inline flows/zeropoint.py:4]: Please give a better docstring!

File: run_plotlc.py [hunk 3]
```diff
@@ -136,6 +146,8 @@ def annotate(sel):
 	mplcursors.cursor(ax).connect("add", annotate)
 	plt.show(block=True)
 
+	if args.savephot != 'None':
```
Comment (rhandberg) [inline run_plotlc.py:149]: Think we should do this in another PR... It has nothing to do with zeropoints and bootstrapping... And it is producing another data product, which I think we should discuss...

**Review by rhandberg — CHANGES_REQUESTED** [review]
Don't think changes to "run_plotlc" belong in this PR. Otherwise only minor changes...

Please run "flake8" and ensure that there are no errors introduced.

File: flows/photometry.py (hunk 1 above)
Comment (emirkmo) [reply flows/photometry.py:647]: It's actually only tracking the difference between the two ZPs, so it shouldn't be confusing. The reason to add them both, for now, is to faciliate comparison. But I'm only tracking the quantity of interest, which is the zp difference. 

The bootstrapped zp is always being used, but I think it would be good to track the "naive" zp as well, at least for monitoring sake.  We want to see if the bootstrapped version is significantly different, and in what scenarios, as it's a bit of a novel/untested way of determining the zp and zp error. I want to study any scenarios where they differ significantly. We may even end up scrapping the bootstrapped version if we find problematic edge cases that we can't resolve.

File: flows/zeropoint.py (hunk 2 above)
Comment (emirkmo) [reply flows/zeropoint.py]: added

Comment (emirkmo) [reply flows/zeropoint.py]: [https://github.com/rhandberg/flows/pull/10/commits/041e327b79fba812876736771836fa1380d32e43](With this commit)

File: run_plotlc.py (hunk 3 above)
Comment (emirkmo) [reply run_plotlc.py]: This is now fixed with this [commit](https://github.com/rhandberg/flows/pull/10/commits/7bde92818dac5058cc071102d5b65464a2913623).

I'll add a separate branch for these changes. I reverted back to the last change made by you.

File: flows/photometry.py (hunk 1 above)
Comment (emirkmo) [reply flows/photometry.py:647]: If you agree, I suggest leaving it as is.

**Comment by rhandberg** [comment]
This PR will also add additional command-line arguments to `run_plotlc` in addition to the original description.
